pinot-common/src/main/java/org/apache/pinot/common/utils/config/TierConfigUtils.java

```diff
@@ -63,6 +67,32 @@ public static String getDataDirForTier(TableConfig tableConfig, String tierName)
     return getDataDirForTier(tableConfig, tierName, Collections.emptyMap());
   }
 
+  /**
+   * Consider configured tiers and compute default instance partitions for the segment
+   *
+   * @return InstancePartitions if the one can be derived from the given sorted tiers, null otherwise
+   */
+  @Nullable
+  public static InstancePartitions getTieredInstancePartitionsForSegment(String tableNameWithType,
+      @Nullable List<Tier> sortedTiers, String segmentName, HelixManager helixManager) {
```

Comment:
nit: adjust the order of params a bit like `String tableNameWithType, String segmentName, ... `

Reply:
Fixed.